src/libraries/System.Net.HttpListener/tests/HttpListenerAuthenticationTests.cs

```diff
@@ -13,8 +13,8 @@
 
 namespace System.Net.Tests
 {
-    [SkipOnCoreClr("System.Net.Tests are flaky")]
-    [SkipOnMono("System.Net.Tests are flaky")]
+    [SkipOnCoreClr("System.Net.Tests may timeout in stress configurations")]
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/2391", TargetFrameworkMonikers.Mono)]
```

Comment:
In order for this to work you need to add an entry to: https://github.com/dotnet/runtime/blob/c2d89a824a7942a8628145930509635c410e2d15/eng/testing/xunit/xunit.targets#L20 like:

```xml
<_targetCategory Condition="'$(RuntimeFlavor)' == 'mono'">mono</_targetCategory>
```

because currently we're not passing the category that active issue discoverer emits for mono TFM:
https://github.com/dotnet/arcade/blob/3c424ae89eec0aa12fed5ad352f3fa44d7c93e92/src/Microsoft.DotNet.XUnitExtensions/src/Discoverers/ActiveIssueDiscoverer.cs#L56

Reply:
Actually never mind, this is not the correct way to do it, we should put it in the mono.rsp file instead, given the way we build the tests and the RunTests.cmd/sh is built at test build and at that point we just build them agnostic of the RuntimeFlavor. Let me dig a little bit more.

Reply:
Ok I just looked around and the .rsp files already have it so it will work just fine:
https://github.com/dotnet/runtime/blob/23c54f183c9faf2049223a44197c6b8b282b90c4/src/mono/netcore/CoreFX.issues.rsp#L4

Reply:
Ok, I was trying to avoid the mono.rsp because there's an open PR to try to move away from it, but I can just update that instead.  I'll push a replacement commit.

Reply:
Unfortunately we have to stick with the .rsp solution for now and leave as is because of the reason I explained above... the RunTests.sh is generated when `RuntimeFlavor == coreclr` in CI because of the way the tests are built, so they won't be skip, let's leave as is and once we remove the .rsp files we can figure out how to do it to keep this change simple.

Reply:
It should just work fine as you have your current changes 😄 -- sorry for the confusion.

Reply:
Oh :(  I just pushed an update to switch to the .rsp... I'll switch back.

Reply:
Sorry about that 😢